src/sentry/static/sentry/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -476,11 +227,151 @@ const AddWidgetWrapper = styled('a')`
-  display: block;
-  position: absolute;
-  cursor: grabbing;
-  opacity: 0.8;
-  box-shadow: ${p => p.theme.dropShadowHeavy};
-  border-radius: ${p => p.theme.borderRadius};
-`;
+type SortableWidgetProps = {
+  widget: Widget;
+  dragId: string;
+  isEditing: boolean;
+  isDragging: boolean;
+  onDelete: () => void;
+  onEdit: () => void;
+};
+
+function SortableWidget(props: SortableWidgetProps) {
+  const {widget, dragId, isDragging, isEditing, onDelete, onEdit} = props;
+
+  const {
+    attributes,
+    listeners,
+    setNodeRef,
+    transform,
+    isDragging: currentWidgetDragging,
+  } = useSortable({
+    id: dragId,
+    transition: null,
+  });
+
+  useEffect(() => {
+    if (!currentWidgetDragging) {
+      return undefined;
+    }
+
+    document.body.style.cursor = 'grabbing';
+
+    return function cleanup() {
+      document.body.style.cursor = '';
+    };
+  }, [currentWidgetDragging]);
+
+  const initialStyles = {
+    x: 0,
+    y: 0,
+    scaleX: 1,
+    scaleY: 1,
+    boxShadow: 'none',
+    zIndex: 0,
+  };
+
+  return (
+    <WidgetWrapper
+      ref={setNodeRef}
+      displayType={widget.displayType}
+      layoutId={dragId}
+      animate={
+        transform
+          ? {
+              x: transform.x,
+              y: transform.y,
+              scaleX: transform?.scaleX && transform.scaleX <= 1 ? transform.scaleX : 1,
+              scaleY: transform?.scaleY && transform.scaleY <= 1 ? transform.scaleY : 1,
+              zIndex: currentWidgetDragging ? theme.zIndex.modal : 0,
+              boxShadow: currentWidgetDragging
+                ? '0 0 0 1px rgba(63, 63, 68, 0.05), 0px 15px 15px 0 rgba(34, 33, 81, 0.25)'
+                : 'none',
+            }
+          : initialStyles
+      }
+      transition={{
+        duration: !currentWidgetDragging ? 0.25 : 0,
+        easings: {
+          type: 'spring',
+        },
+        transform: {duration: 0},
+        scale: {
+          duration: 0.25,
+        },
+        zIndex: {
+          delay: currentWidgetDragging ? 0 : 0.25,
+        },
+      }}
+    >
+      <WidgetCard
+        widget={widget}
+        isEditing={isEditing}
+        onDelete={onDelete}
+        onEdit={onEdit}
+        isDragging={isDragging}
+        hideToolbar={isDragging}
+        currentWidgetDragging={currentWidgetDragging}
+        draggableProps={{
+          attributes,
+          listeners,
+        }}
+      />
+    </WidgetWrapper>
+  );
+}
+
+function AddWidget(props: {onClick: () => void}) {
+  const {onClick} = props;
+
+  const initialStyles = {
+    x: 0,
+    y: 0,
+    scaleX: 1,
+    scaleY: 1,
+    zIndex: 0,
+  };
+
+  const {setNodeRef, transform} = useSortable({
+    disabled: true,
+    id: ADD_WIDGET_BUTTON_DRAG_ID,
+    transition: null,
+  });
+
+  return (
+    <WidgetWrapper
+      key="add"
+      ref={setNodeRef}
+      displayType="big_number"
+      layoutId={ADD_WIDGET_BUTTON_DRAG_ID}
+      animate={
+        transform
```

Comment:
If the add button isn't draggable do you need this?

Reply:
The add button isn't a draggable, but it's a droppable. I'll revisit to ensure if it needs to be a `<motion.div/>`.